apps/console/src/public/auth-spa-0.3.3.min.js

```diff
@@ -0,0 +1,2 @@
IsIlJlcXVlc3QgdG8gdGhlIHByb3ZpZGVkIGVuZHBvaW50IGlzIHByb2hpYml0ZWQuIiwiUmVxdWVzdHMgY2FuIG9ubHkgYmUgc2VudCB0byByZXNvdXJjZSBzZXJ2ZXJzIHNwZWNpZmllZCBieSB0aGUgYHJlc291cmNlU2VydmVyVVJMc2AgYXR0cmlidXRlIHdoaWxlIGluaXRpYWxpemluZyB0aGUgU0RLLiBUaGUgc3BlY2lmaWVkIGVuZHBvaW50IGluIHRoaXMgcmVxdWVzdCBjYW5ub3QgYmUgZm91bmQgYW1vbmcgdGhlIGByZXNvdXJjZVNlcnZlclVSTHNgIikpfSkpLGh0dHBSZXF1ZXN0QWxsOnI9PmUodm9pZCAwLHZvaWQgMCx2b2lkIDAsKGZ1bmN0aW9uKigpe3ZhciBuLGksbztsZXQgdT0hMDtmb3IoY29uc3QgZSBvZiByKXtsZXQgcj0hMTtmb3IoY29uc3QgYSBvZlsuLi5udWxsIT09KGk9bnVsbD09PShuPXlpZWxkIHMuZ2V0Q29uZmlnRGF0YSgpKXx8dm9pZCAwPT09bj92b2lkIDA6bi5yZXNvdXJjZVNlcnZlclVSTHMpJiZ2b2lkIDAhPT1pP2k6W10sdC5iYXNlVXJsXSlpZihhJiYobnVsbD09PShvPWUudXJsKXx8dm9pZCAwPT09bz92b2lkIDA6by5zdGFydHNXaXRoKGEpKSl7cj0hMDticmVha31pZighcil7dT0hMTticmVha319Y29uc3QgaD1bXTtyZXR1cm4gdT8oci5mb3JFYWNoKChlPT57aC5wdXNoKGMucmVxdWVzdChlKSl9KSksKG51bGw9PWM/dm9pZCAwOmMuYWxsKSYmYy5hbGwoaCkudGhlbigoZT0+UHJvbWlzZS5yZXNvbHZlKGUpKSkuY2F0Y2goKHQ9PmUodm9pZCAwLHZvaWQgMCx2b2lkIDAsKGZ1bmN0aW9uKigpe3ZhciBlLHIsbjtpZig0MDE9PT0obnVsbD09PShlPW51bGw9PXQ/dm9pZCAwOnQucmVzcG9uc2UpfHx2b2lkIDA9PT1lP3ZvaWQgMDplLnN0YXR1cykpe2xldCBlO3RyeXtlPXlpZWxkIGwoKX1jYXRjaChlKXt0aHJvdyBuZXcgYSgiU1BBLVdPUktFUl9DT1JFLUhSQS1TRTAxIixudWxsIT09KHI9bnVsbD09ZT92b2lkIDA6ZS5uYW1lKSYmdm9pZCAwIT09cj9yOiJSZWZyZXNoIHRva2VuIHJlcXVlc3QgZmFpbGVkLiIsbnVsbCE9PShuPW51bGw9PWU/dm9pZCAwOmUubWVzc2FnZSkmJnZvaWQgMCE9PW4/bjoiQW4gZXJyb3Igb2NjdXJyZWQgd2hpbGUgdHJ5aW5nIHRvIHJlZnJlc2ggdGhlIGFjY2VzcyB0b2tlbiBmb2xsb3dpbmcgYSA0MDEgcmVzcG9uc2UgZnJvbSB0aGUgc2VydmVyLiIpfWlmKGUpcmV0dXJuIGMuYWxsJiZjLmFsbChoKS50aGVuKChlPT5Qcm9taXNlLnJlc29sdmUoZSkpKS5jYXRjaCgoZT0+UHJvbWlzZS5yZWplY3QoZSkpKX1yZXR1cm4gUHJvbWlzZS5yZWplY3QodCl9KSkpKSk6UHJvbWlzZS5yZWplY3QobmV3IGEoIlNQQS1XT1JLRVJfQ09SRS1IUkEtSVYwMiIsIlJlcXVlc3QgdG8gdGhlIHByb3ZpZGVkIGVuZHBvaW50IGlzIHByb2hpYml0ZWQuIiwiUmVxdWVzdHMgY2FuIG9ubHkgYmUgc2VudCB0byByZXNvdXJjZSBzZXJ2ZXJzIHNwZWNpZmllZCBieSB0aGUgYHJlc291cmNlU2VydmVyVVJMc2AgYXR0cmlidXRlIHdoaWxlIGluaXRpYWxpemluZyB0aGUgU0RLLiBUaGUgc3BlY2lmaWVkIGVuZHBvaW50IGluIHRoaXMgcmVxdWVzdCBjYW5ub3QgYmUgZm91bmQgYW1vbmcgdGhlIGByZXNvdXJjZVNlcnZlclVSTHNgIikpfSkpLGlzQXV0aGVudGljYXRlZDooKT0+aS5pc0F1dGhlbnRpY2F0ZWQoKSxyZWZyZXNoQWNjZXNzVG9rZW46bCxyZXF1ZXN0QWNjZXNzVG9rZW46KHQscixuLGMpPT5lKHZvaWQgMCx2b2lkIDAsdm9pZCAwLChmdW5jdGlvbiooKXtjb25zdCBlPXlpZWxkIHMuZ2V0Q29uZmlnRGF0YSgpO3JldHVybiBuJiZlLmVuYWJsZVBLQ0UmJih5aWVsZCBpLnNldFBLQ0VDb2RlKG4sbnVsbCE9Yz9jOiIiKSksdD9pLnJlcXVlc3RBY2Nlc3NUb2tlbih0LG51bGwhPXI/cjoiIixudWxsIT1jP2M6IiciKS50aGVuKCgoKT0+KG8ucmVmcmVzaEFjY2Vzc1Rva2VuQXV0b21hdGljYWxseSgpLGkuZ2V0QmFzaWNVc2VySW5mbygpKSkpLmNhdGNoKChlPT5Qcm9taXNlLnJlamVjdChlKSkpOlByb21pc2UucmVqZWN0KG5ldyBhKCJTUEEtV09SS0VSX0NPUkUtUkFUMS1ORjAxIiwiTm8gYXV0aG9yaXphdGlvbiBjb2RlLiIsIk5vIGF1dGhvcml6YXRpb24gY29kZSB3YXMgZm91bmQuIikpfSkpLHJlcXVlc3RDdXN0b21HcmFudDp1LHJldm9rZUFjY2Vzc1Rva2VuOigpPT5pLnJldm9rZUFjY2Vzc1Rva2VuKCkudGhlbigoKCk9PihvLmNsZWFyUmVmcmVzaFRva2VuVGltZW91dCgpLFByb21pc2UucmVzb2x2ZSghMCkpKSkuY2F0Y2goKGU9PlByb21pc2UucmVqZWN0KGUpKSksc2V0SHR0cFJlcXVlc3RGaW5pc2hDYW
```

Comment:
Shall we create an issue to see if we can create a GitHub CI cronjob to automatically update this once a new release is done in the SPA SDK?

Reply:
This is tracked with https://github.com/wso2-enterprise/asgardeo-product/issues/11666